terms.html

```diff
@@ -154,6 +171,27 @@
 by a <a>DID fragment</a>.
   </dd>
 
+  <dt><dfn>DID URL dereferencing</dfn></dt>
+
+  <dd>
+The function that takes as its input a <a>DID URL</a>, a <a>DID document</a>,
+and a set of dereferencing options and returns a resource, which might be
+a <a>DID document</a>, plus additional metadata. This function can use the <a>DID resolution</a>
+function to fetch a <a>DID document</a> indicated by the <a>DID</a> within the <a>DID URL</a>.
+The dereferencing function then can perform additional processing on the <a>DID document</a> to return
+the dereferenced resource indicated by the <a>DID URL</a>. The contract of this
+function is defined in <a href="#did-url-dereferencing"></a>.
+  </dd>
+
+  <dt><dfn data-lt"DID URL dereferencers">DID URL dereferencer</dfn></dt>
+
+  <dd>
+A software and/or hardware system that is capable of executing the <a>DID URL dereferencing</a>
+function, retrieving some form of data, which might or might not be a <a>DID document</a>) for a given
+<a>DID URL</a>. The details of systems are specified in the DID Resolution specification
+[[?DID-RESOLUTION]].
```

Comment:
Suggest we remove the outbound reference here and define component in terms of the function it implements.

```suggestion
A software and/or hardware system that is capable of executing the <a>DID URL dereferencing</a>
function, retrieving some form of data, which might or might not be a <a>DID document</a>) for a given
<a>DID URL</a> by performing <a>DID URL dereferencing</a>.
```